src/dqe_fun.erl

```diff
@@ -10,9 +10,12 @@
 
 -export([init/0, reg/1, lookup/2, list/0, render/1]).
 -export_type([spec/0]).
+
+-ignore_xref([behaviour_info/1, list/0, lookup/2, reg/1, render/1]).
+
 -define(TBL, dqe_fun).
 
--type return_type() :: histogram | metric.
+-type return_type() :: histogram | metric | time.
```

Comment:
I originally thought the same, but the change I made in the parser used `time` as the return:
```

time(T, U) ->
    #{
      op => time,
      args => [T, U],
      signature => [integer, time_unit],
      return => time
     }.
```
Does the return actually refer to the type itself then?